OKR.md

```diff
@@ -2,6 +2,27 @@
 
 We try to frame our ongoing work using a process based on quarterly Objectives and Key Results (OKRs). Objectives reflect outcomes that are challenging, but realistic. Results are tangible and measurable.
 
+## 2018 Q4
+
+**Debuggability and Stability**
+- error handling is done correctly and error codes are part of the API - @jacobheun
+- libp2p has a network visualization tool - @jacobheun
+- js-libp2p has a daemon with a RPC API and a client library - @jacobheun
+- interop tests for go-libp2p and js-libp2p exist for core functionality - @vasco-santos
+- a testbed exists and can test 50,000 libp2p nodes (browser and node) - @jacobheun
+- the testbed can run large scale tests for the DHT and Pubsub - @vasco-santos
+
+**Connectivity Magic is better and more reliable**
+- the DHT is enabled by default in js-ipfs and is interoperable with go - @vasco-santos
+- IPNS works over DHT and is interoperable with go - @vasco-santos
+- IPNS spec exists and is complete - @vasco-santos
+- Nat traversal Spec is ready for js-libp2p to implement - @jacobheun
+
+**Libp2p is seen as the go to p2p networking stack**
+- Usage examples and proper documentation exist for the key modules of libp2p - @jacobheun
+
+Once these OKRs are finished, you will be able to find them on the [2018 Q4 libp2p OKRs Spreadsheet](https://docs.google.com/spreadsheets/d/1BYwmbVicgo6_tOHAbgiUXWge8Ej0qR1M_gAUulazmrg/edit#gid=1241853194)
+
```

Comment:
It would be good now to do a Timing Exercise (count the number of days that you effectively have in the quarter) and plan for 60% of it. 

At the same time, add priorities for each of these KRs.

Reply:
For me: 66 total days, minus scheduled holidays and team week (53 days), for a dev capacity at 60% of **31.8 days.**

This gives us just over 6weeks each of dev time. 

I added priorities to the items, let me know your thoughts. 

As it stands, with our availability and the priority order, I think realistically the `network visualization tool`, `nat traversal spec` and `error handling` won't be completed. I think error handling will improve for code we'll be working in, but additional efforts to improve existing code will take a back seat.  I think these are good for stretch goals though.